source/guide/integrate-to-your-app.md

```diff
@@ -25,7 +25,7 @@ The keys to intergrating Weex into your Android application are the following fi
 dependencies {
     ...
     // weex sdk and fastjson
-    compile 'com.taobao.android:weex_sdk:0.5.1@aar'
+    compile 'com.taobao.android:weex_sdk:0.10.0@aar'
```

Comment:
Good catch! But I think the v0.10.0 is also an old one. According to the [release note](http://weex-project.io/releasenote.html), v0.18.0 is the latest for now. Change it to `0.18.0` is more reasonable.

Reply:
Changed to 0.18.0 and verified. Works fine